notifications_utils/timezones.py

```diff
@@ -1,32 +1,44 @@
-from datetime import datetime
+import os
 
 import pytz
 from dateutil import parser
 
-local_timezone = pytz.timezone("Europe/London")
+local_timezone = pytz.timezone(os.getenv("TIMEZONE", "America/New_York"))
 
 
 def utc_string_to_aware_gmt_datetime(date):
     """
-    Date can either be a string, naive UTC datetime or an aware UTC datetime
-    Returns an aware London datetime, essentially the time you'd see on your clock
+    Date can either be a string, naïve UTC datetime or an aware UTC datetime
+    Returns an aware local datetime, essentially the time you'd see on your clock
     """
-    if not isinstance(date, datetime):
-        date = parser.parse(date)
-
+    date = parser.parse(date)
     forced_utc = date.replace(tzinfo=pytz.utc)
     return forced_utc.astimezone(local_timezone)
 
 
-def convert_utc_to_bst(utc_dt):
+def convert_utc_to_est(utc_dt):
```

Comment:
[suggestion] can we just get rid of the to/from `est` versions in favor of the new `local_timezone` one? Either that, or let's hardcode the use of `America/New_York` in these two. I think it would be super confusing to call `convert_utc_to_est` and get back a different timezone if `env["TIMEZONE"]` was set.

Reply:
Yep, we can get rid of it. I was holding off until I tackled the UI, but it looks like it's not used there, so we should be good.